wiki/Article_Styling_Criteria/en.md

```diff
@@ -82,150 +100,232 @@ tags:
 
 ### Folder names
 
-Folder names must never contain URL reserved characters. Generally, all folder names should only use these characters:
+Folder names must never contain URL reserved characters. Generally, all folder names must use these characters:
 
 - A - Z (uppercase letters)
 - a - z (lowercase letters)
 - 0 - 9 (numbers)
```

Comment:
These characters don't have the code markup. The three lines shown.

Reply:
These are just talking about letters and numbers, not keyboard keys.

Reply:
The code markup is used on letters or numbers on various places of this article. They are used for emphasizing a certain thing in writing. In this case, it's characters and how you only use these characters in naming folders.